rfc3164/message_test.go

```diff
@@ -37,7 +37,7 @@ func (s *Rfc3164MessageTestSuite) TestMessageFromRfc3164(c *C) {
   c.Assert(string(sampleRfc3164Log), Equals, string(*msg.RawMessage()))
   c.Assert("webtest-mark", Equals, msg.Hostname())
   c.Assert("This is a log.info() message", Equals, msg.Message())
-  c.Assert(true, Equals, time.Unix(1402085235, 0).Equal(msg.TimeStamp()))
+  c.Assert(true, Equals, time.Unix(1433621235, 0).Equal(msg.TimeStamp()))
```

Comment:
This if for a patch release. On master the year is being set, but this requires a refactoring of the system under test in order to set the year to a constant year and we're trying to minimize the number of changes pulled in.